website/docs/guides/relay-resolvers/context.md

```diff
@@ -0,0 +1,120 @@
+---
+id: context
+title: "Context"
+slug: /guides/relay-resolvers/context
+description: Context in Relay Resolvers
+---
+import {FbInternalOnly, fbContent} from 'docusaurus-plugin-internaldocs-fb/internal';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+In order to pass a service, or other values to be shared with all resolvers, the `LiveResolverStore` provides a means of passing context. This gets passed to the third argument of all resolvers (live and non-live).
+
+## Setup
+
+In order to pass context to live resolvers, pass a `resolverContext` argument to the initialization of `LiveResolverStore` before creating the environment:
+
+```js
+const store = new LiveResolverStore(source, {
+  resolverContext: {
+    store: customStore,
+  },
+});
+```
+
+## Usage in Resolvers
+
+<Tabs
+  groupId="resolver"
+  defaultValue="Docblock"
+  values={fbContent({
+    internal: [
+      {label: 'Docblock', value: 'Docblock'},
+      {label: 'Flow', value: 'Flow'},
+    ],
+    external: [
+      {label: 'Docblock', value: 'Docblock'},
+        {label: 'Flow', value: 'Flow'},
+    ]
+  })}>
+  <TabItem value="Docblock">
+
+The third argument in a resolver will contain the context type which contains the value passed into the store on initialization.
+
+```tsx
+import type { LiveState } from 'relay-runtime';
+
+/**
+ * @RelayResolver Query.counter: Int
+ * @live
+ */
+export function counter(
+  _key: undefined, 
+  _args: undefined, 
+  context: IResolverContextType
+): LiveState<number> {
+  return {
+    read: () => context.store.getState().counter,
+    subscribe: (callback) => {
+      return context.store.subscribe(callback);
+    },
+  };
+}
+```
+
+  </TabItem>
+
+  <TabItem value="Flow">
```

Comment:
Context won't work with the Flow syntax until we do some additional work. Can you replace these Flow tabs with a note saying it's not supported for now?